pyiceberg/catalog/glue.py

```diff
@@ -150,7 +152,7 @@ def primitive(self, primitive: PrimitiveType) -> str:
         if isinstance(primitive, DecimalType):
             return f"decimal({primitive.precision},{primitive.scale})"
         if (primitive_type := type(primitive)) not in GLUE_PRIMITIVE_TYPES:
-            raise ValueError(f"Unknown primitive type: {primitive}")
+            return str(primitive_type.root)
```

Comment:
Since this is only for the StorageDescriptor, which is optional, I think it should not block the table creation even when encountering unknown primitive types